### ISSUE TITLE
Default parallelism is set to 1

### ISSUE THREAD

**Issue by carterwilliamson** [pulumi/pulumi#2108]
Pulumi CLI version: v0.16.1
@pulumi/pulumi version: v0.16.1

**Expected**
Running `pulumi up` should be executing with `parallelism = 0`

**Actual**
Running `pulumi up` appears to be executing with `parallelism = 1`

**Comment by sylv3r** [comment]
Ok, so basically you changed the default behavior of the program without warning on a minor update...

Seriously ?

**Comment by lukehoban** [comment]
@sylv3r this was definitely a bug, not an intentional breaking change. From investigation so far, this will only happen if you use the new `pulumi` CLI along with an older version of `@pulumi/pulumi`. Unfortunately, that older version can come indirectly from a transitive dependency, and some common upgrade paths will by default lead to this combination. 

You should be able to fix this with `npm upgrade` after moving your package.json dependencies to use `@pulumi/pulumi` 0.6.1. 

Very sorry for the trouble here. We will be adding new test coverage to ensure we can catch issues related to upgrade paths like this in the future.

**Comment by joeduffy** [comment]
@swgillespie Is it possible to fix the CLI and release a new build that doesn't suffer from this problem? Also /cc @ellismg for the logistics of releasing a new point release.

**Comment by swgillespie** [comment]
> @swgillespie Is it possible to fix the CLI and release a new build that doesn't suffer from this problem?

Yes, I can do this.

**Comment by lukehoban** [comment]
Yes - let’s work around the SDK bug by releasing a fix in the CLI for now as that provides the easiest path for users to get unblocked. We should then also fix the SDK.

**Comment by ellismg** [comment]
0.16.2 has been released and it fixes this issue.  You can install it with `curl -sL https://get.pulumi.com/ | bash -s -- --version 0.16.2` and we'll be updating pulumi.io to point to the newer version today.
